src/uwtools/config/support.py

```diff
@@ -12,6 +14,20 @@
 INCLUDE_TAG = "!INCLUDE"
 
 
+# Public functions
+def add_representers():
+    """
+    Add representers to the YAML dumper for custom types.
+
+    This function is called by the YAMLConfig class to add representers for the TaggedString,
+    Namelist, and OrderedDict types to the YAML dumper. The representers are used to serialize these
+    types to YAML.
+    """
```

Comment:
I'd recommend not stating in the docstring what code might call this code -- any code could call it. The docstring should only describe the behavior of _this_ function.